src/XRP/rippled-flags.ts

```diff
@@ -8,6 +8,8 @@
 class RippledFlags {
   static TF_PARTIAL_PAYMENT = 1 << 17
 
+  // https://xrpl.org/accountroot.html#accountroot-flags
```

Comment:
I think that because this is a separate flag sets it should live in a different file